### DISCUSSION TITLE
Ignite cli Inter-Blockchain Communication

### DISCUSSION THREAD

**Discussion by priyanshuthapliyal** [cosmos/community#54] [Chain Builders (Cosmos SDK)] [answered]
I am following this docs to create two chain and send transaction between those two:
https://docs.ignite.com/guide/ibc
NOw when creating confige file for both the chains i am getting this error

 ignite chain serve -c earth.yml
Cosmos SDK's version is: stargate - v0.45.4

config is not valid: validator is required

is in the docs is there something missing?

**Comment by tac0turtle** [comment, marked as answer]
hey sorry for the long delay, you would need to ask the ignite team directly, we dont maintain that software
